apps/web/components/IcelandicNamesSearcher/IcelandicNamesSearcher.tsx

```diff
@@ -205,228 +215,264 @@ export const IcelandicNamesSearcher: FC = () => {
-                    />
-                    <Checkbox
-                      label="Millinöfn (öll kyn)"
-                      checked={filters.middleNames}
-                      onChange={() => dispatch({ type: 'toggleMiddleNames' })}
-                    />
-                    {!!typeFilterSelected && (
-                      <ResetButton
-                        onClick={() => dispatch({ type: 'clearCategories' })}
-                      >
-                        Hreinsa val
-                      </ResetButton>
-                    )}
-                  </Stack>
-                </SidebarAccordion>
-                <Box paddingY={2}>
-                  <Divider />
-                </Box>
-                <SidebarAccordion id="icelandic_names_statuses" label="Staða">
-                  <Stack space={[1, 1, 2]}>
-                    <Checkbox
-                      label="Samþykkt"
-                      checked={filters.approved}
-                      onChange={() => dispatch({ type: 'toggleApproved' })}
-                    />
-                    <Checkbox
-                      label="Hafnað"
-                      checked={filters.denied}
-                      onChange={() => dispatch({ type: 'toggleDenied' })}
-                    />
-                    {!!statusFilterSelected && (
-                      <ResetButton
-                        onClick={() => dispatch({ type: 'clearStatuses' })}
-                      >
-                        Hreinsa val
-                      </ResetButton>
-                    )}
-                  </Stack>
-                </SidebarAccordion>
-              </Box>
-              {!!someFilterSelected && (
-                <ResetButton onClick={() => dispatch({ type: 'clearAll' })}>
-                  Hreinsa síu
+                  Hreinsa val
                 </ResetButton>
               )}
             </Stack>
-          </GridColumn>
+          </SidebarAccordion>
+          <Box paddingY={2}>
+            <Divider />
+          </Box>
+          <SidebarAccordion id="icelandic_names_statuses" label="Staða">
+            <Stack space={[1, 1, 2]}>
+              <Checkbox
+                label="Samþykkt"
+                checked={filters.approved}
+                onChange={() => dispatch({ type: 'toggleApproved' })}
+              />
+              <Checkbox
+                label="Hafnað"
+                checked={filters.denied}
+                onChange={() => dispatch({ type: 'toggleDenied' })}
+              />
+              {!!statusFilterSelected && (
+                <ResetButton
+                  onClick={() => dispatch({ type: 'clearStatuses' })}
+                >
+                  Hreinsa val
+                </ResetButton>
+              )}
+            </Stack>
+          </SidebarAccordion>
+        </Box>
+        {!!someFilterSelected && (
+          <ResetButton onClick={() => dispatch({ type: 'clearAll' })}>
+            Hreinsa síu
```

Comment:
A little above you had the text `Hreinsa síu` but changed it to `Hreinsa val`.
Is this inconsistent?

Reply:
Good point.